packages/react-dom/src/test-utils/ReactTestUtils.js

```diff
@@ -711,6 +727,23 @@ function buildSimulators() {
 }
 buildSimulators();
 
+let didWarnAboutUsingAct = false;
+export const act = __DEV__
```

Comment:
Should we also fork dev and prod on `main`? We always warn on `main`.

Reply:
but it throws in production anyway so it's not a big deal